src/shared/components/VideoTileBase/VideoTileBase.tsx

```diff
@@ -155,16 +150,22 @@ export const VideoTileBase: React.FC<VideoTileBaseProps> = ({
   isPullupDisabled,
 }) => {
   const { openContextMenu, contextMenuOpts } = useContextMenu()
-  const [scalingFactor, setScalingFactor] = useState(MIN_SCALING_FACTOR)
+  const [tileSize, setTileSize] = useState<TileSize>(undefined)
+
   const { ref: imgRef } = useResizeObserver<HTMLImageElement>({
     onResize: (size) => {
       const { width: videoTileWidth, height: videoTileHeight } = size
+      if (videoTileWidth && !main) {
+        if (tileSize !== 'small' && videoTileWidth <= SMALL_SIZE_WIDTH) {
+          setTileSize('small')
+        }
+        if (tileSize !== 'big' && videoTileWidth > SMALL_SIZE_WIDTH) {
```

Comment:
Let's change the inclusivity of this condition and follow mobile-first. So if width === 300 let's already use the bigger version (I know design says otherwise but we talked about this with designers and this should be approach we're using)